examples/cfhtls_wide_to_astrometry_net.py

```diff
@@ -1,7 +1,5 @@
 from __future__ import print_function
```

Comment:
In theory you could remove this as well.

Reply:
Good suggestions. I'll do it in another commit.